Makefile

```diff
@@ -108,8 +108,9 @@ else
 	$(call MINIKUBE_GO_BUILD_CMD,$@,$*)
 endif
 
-out/e2e-%-amd64:
-	GOOS=$* GOARCH=amd64 go test -c k8s.io/minikube/test/integration --tags=integration -o $@
+.PHONY: e2e-%-amd64
+e2e-%-amd64:
```

Comment:
We use this rule in hack/jenkins/cross-build now to generate the e2e binaries, so there are times when we want to build it, but not necessarily run it.    

I don't think you could actually run `sudo go test ...` anways, so you need this intermediate binary for the none driver.

I think this should recompile each time since its a PHONY target right?